### DISCUSSION TITLE
点击节点会触发拖拽事件，可以禁止拖拽事件或者延迟触发吗？

### DISCUSSION THREAD

**Discussion by dustandsnow** [didi/LogicFlow#1881] [Q&A]
![TIM_IMG_20240920173604](https://github.com/user-attachments/assets/1e7f7a0a-73a2-4f4b-ab87-5654e70613ea)
点击节点时间很短还是会触发拖拽事件，有没有方式在点击节点时不触发拖拽事件

**Comment by EdwinOlders11** [comment]
使用html节点并在拖曳时阻止事件冒泡实测可以阻止鼠标点击时触发拖拽事件的问题
